### PR TITLE
Add desktop version

### PR DESCRIPTION
**Pull request by Botlhale-Setou**
Add desktop `<nav>` section w/ corresponding links.
Add desktop .headline section, w/ hover effect.
Add desktop .work section w/ corresponding .png files.
Add desktop backgrounds.
Ensure running of desktop and mobile versions.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -74,25 +102,28 @@ <h2 class="supporting-text">Software Developer</h2>
...
+        
       </div>
     </section>
 
     <section class="work">
       <header class="work-header">
-        <p class="work-header-title">MY RECENT WORK</p>
-        <img class="work-divider" src="./media/Divider.png" alt="A content divider">
+        <p class="work-header-title m">MY RECENT WORK</p>
+        <p class="work-header-title d">My recent work</p>
```
Comment (GzimAsani) [inline index.html]: - [x] Because of SEO reasons, this needs be an hx tag, and it should not be bigger than your current h1 tag

File: index.html [hunk 2]
```diff
@@ -74,25 +102,28 @@ <h2 class="supporting-text">Software Developer</h2>
...
         <div class="entry">
-          <div class="img-holder">
+          <div class="img-holder to-left">
             <img class="entry-img" src="#" alt="Recent work snapshot">
           </div>
-          <section class="entry-data">
+          <section class="entry-data to-right">
             <p class="entry-title">Multi-Post Stories</p>
             <p class="entry-text">A daily selection of privately personalized reads; 
               no accounts or sign-ups required. has been the industry's standard
```
Comment (GzimAsani) [inline index.html:129]: - [x] These technologies need to be centered, kindly check the design
image

File: index.html [hunk 3]
```diff
@@ -182,9 +213,12 @@ <h2 class="supporting-text">Software Developer</h2>
     <footer class="form-footer">
       <form action="#" method="post">
         <div>
-          <h2 class="contact-title">
+          <h2 class="contact-title m">
             CONTACT US
           </h2>
+          <h2 class="contact-title d">
+            Let's chat!
```
Comment (GzimAsani) [inline index.html:220]: - [x] Kindly add the placeholders as in the design
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @Botlhale-Setou   :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job  with the styling, looks good  :+1: 
 
 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 




 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Botlhale-Setou) [reply index.html]: @GzimAsani Change applied 👍

File: index.html (hunk 2 above)
Comment (Botlhale-Setou) [reply index.html:129]: @GzimAsani Changes made 👍

File: index.html (hunk 3 above)
Comment (Botlhale-Setou) [reply index.html:220]: @GzimAsani Changes made 👍

**Review by ghost — APPROVED** [review]
Hi @Botlhale-Setou , 

Good work implementing the changes requested. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)

## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Good Design 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**